precompiles/assets-erc20/src/lib.rs

```diff
@@ -522,4 +546,406 @@ where
 			logs: Default::default(),
 		})
 	}
+
+	// From here: only for locals, we need to check whether we are in local assets otherwise fail
+	fn mint(
+		asset_id: AssetIdOf<Runtime, Instance>,
+		input: &mut EvmDataReader,
+		gasometer: &mut Gasometer,
+		context: &Context,
+	) -> EvmResult<PrecompileOutput> {
+		if !IsLocal::get() {
+			return Err(gasometer.revert("unknown selector"));
+		}
+
+		gasometer.record_log_costs_manual(3, 32)?;
+
+		// Parse input.
+		input.expect_arguments(gasometer, 2)?;
+
+		let to: H160 = input.read::<Address>(gasometer)?.into();
+		let amount = input.read::<BalanceOf<Runtime, Instance>>(gasometer)?;
+
+		// Build call with origin.
+		{
+			let origin = Runtime::AddressMapping::into_account_id(context.caller);
+			let to = Runtime::AddressMapping::into_account_id(to);
+
+			// Dispatch call (if enough gas).
+			RuntimeHelper::<Runtime>::try_dispatch(
+				Some(origin).into(),
+				pallet_assets::Call::<Runtime, Instance>::mint {
+					id: asset_id,
+					beneficiary: Runtime::Lookup::unlookup(to),
```

Comment:
Is this lookup accounted for with the gasometer?

Reply:
(there are other instances of the same that follow)

Reply:
AFAIK, this look up does not involve any DB write or read https://github.com/paritytech/substrate/blob/master/primitives/runtime/src/traits.rs. Therefore it is not accounted